src/WSRequestHandler_Sources.cpp

```diff
@@ -315,6 +315,97 @@ RpcResponse WSRequestHandler::SetVolume(const RpcRequest& request)
 	return request.success();
 }
 
+/**
+* Changes whether an audio track is active for a source.
+*
+* @param {String} `source` Source name.
```

Comment:
It'd be better to use the field `sourceName` instead of `source` to be more consistent with our current naming system.